course_catalog/etl/see.py

```diff
@@ -0,0 +1,227 @@
+"""Sloan course catalog ETL"""
+import logging
+import re
+from datetime import datetime
+from decimal import Decimal
+from urllib.parse import urljoin
+
+import requests
+from bs4 import BeautifulSoup as bs
+from django.conf import settings
+
+from course_catalog.constants import OfferedBy, PlatformType
+from course_catalog.etl.utils import (
+    log_exceptions,
+    generate_unique_id,
+    strip_extra_whitespace,
+    parse_dates,
+)
+
+OFFERED_BY = [{"name": OfferedBy.see.value}]
+
+log = logging.getLogger()
+
+
+def _parse_topics(course):
+    """
+    Extract topics for a course
+
+    Args:
+        course(Tag): The BeautifulSoup tag containing course data
+
+    Returns:
+          list of str: List of topic names
+
+    """
+    topics = []
+    for classname in ("primary-topics", "secondary-topics"):
+        try:
+            topics.extend(
+                [
+                    strip_extra_whitespace(topic)
+                    for topic in course.find("span", {"class": classname})
+                    .parent.find(text=True, recursive=False)
+                    .split(",")
+                ]
+            )
+        except AttributeError:
+            pass
+    return topics
+
+
+def _parse_price(course):
+    """
+    Extract the course price as a list
+
+    Args:
+        course(Tag): The BeautifulSoup tag containing course data
+
+    Returns:
+          list of int: List containing 1 integer (or empty)
+    """
+    price = None
+    try:
+        price_text = course.find("strong", text=re.compile(r"Tuition:.*")).parent.find(
+            text=True, recursive=False
+        )
+    except AttributeError:
+        return price
```

Comment:
We can just `return None` here since we never set `price` to anything in the function